ui/components/app/snaps/update-snap-permission-list/update-snap-permission-list.stories.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import UpdateSnapPermissionList from './update-snap-permission-list';
 
 export default {
-  title: 'Components/App/UpdateSnapPermissionList',
+  title: 'Components/App/Snaps/UpdateSnapPermissionList',
   component: UpdateSnapPermissionList,
 
```

Comment:
as the `targetedSubjectMetadata` is not passed in the story it throws an error in console
can the object be passed as the one as pasted below?
```
{
    name: PropTypes.string,
    origin: PropTypes.string.isRequired,
    subjectType: PropTypes.string.isRequired,
    extensionId: PropTypes.string,
    iconUrl: PropTypes.string,
}
```